docs/source/reference/logging.rst

```diff
@@ -0,0 +1,200 @@
+Logging
+=======
+
+To log with Perceval you can either a built-in Perceval logger or the python one. By default, our logger will be used.
+
+To have the perceval message log with the python logger, use this method:
```

Comment:
To log the Perceval message logs with the python logger, use this method:

(the word log gets too repetitive I understand but 'have' seemed weird to me, maybe 'save'?)

Reply:
To log a message with the Python logger, use: